cmd/loadTests.go

```diff
@@ -310,38 +314,54 @@ func setup(cmd *cobra.Command, args []string) {
 
 	logData.LoadTestCompletionStatus = "Completed"
 
-	averageTimeToSpinUpUsers := averageDurationFromArray(AverageUserCreationTime)
+	userCreationFailureCount := sumFromArray(FailedUserCreations)
+	logData.UserCreationFailureCount = userCreationFailureCount
+
+	averageTimeToSpinUpUsers := float64(0)
+	if userCreationFailureCount < int64(overallCount) {
+		averageTimeToSpinUpUsers = sumDurationFromArray(AverageUserCreationTime).Seconds() / float64(int64(overallCount)-userCreationFailureCount)
+	}
 	logData.AverageTimeToSpinUpUsers = averageTimeToSpinUpUsers
 
-	averageTimeToCreateResources := averageDurationFromArray(AverageResourceCreationTimePerUser)
+	resourceCreationFailureCount := sumFromArray(FailedResourceCreations)
+	logData.ResourceCreationFailureCount = resourceCreationFailureCount
+
+	averageTimeToCreateResources := float64(0)
+	if resourceCreationFailureCount < int64(overallCount) {
+		averageTimeToCreateResources = sumDurationFromArray(AverageResourceCreationTimePerUser).Seconds() / float64(int64(overallCount)-resourceCreationFailureCount)
```

Comment:
Hello, I do not know how this works, but lets say we run for 10 users, one hread, so 

`overallCount = 10`

Let's say 2 users fail creation and out of these 8 applications, 2 fails.

Is `resourceCreationFailureCount` 2 or 4 now?

Asking because of this code that counts number of correctly created (e.g.) applications:

`int64(overallCount)-resourceCreationFailureCount`

Reply:
@jhutar Good catch, thx. I've refactored a bit the part to count successful steps correctly.